rpcserver.go

```diff
@@ -3412,12 +3419,12 @@ func marshalOutboundParcel(
 func marshalOutputProofDeliveryStatus(
 	out tapfreighter.TransferOutput) taprpc.ProofDeliveryStatus {
 
-	proofDeliveryStatus := taprpc.ProofDeliveryStatusNotApplicable
+	proofDeliveryStatus := rpcutils.ProofDeliveryStatusNotApplicable
 	out.ProofDeliveryComplete.WhenSome(func(complete bool) {
 		if complete {
-			proofDeliveryStatus = taprpc.ProofDeliveryStatusComplete
+			proofDeliveryStatus = rpcutils.ProofDeliveryStatusComplete //nolint: lll
```

Comment:
nit: add nolint comment before `if` so it counts for both branches and doesn't make line even longer?